resources/checks/checks.yml

```diff
@@ -483,6 +495,7 @@ post:
           checkEditable.setFrom("adr_210a8d4b0b76d77b");
           checkEditable.setTo(to);
           checkEditable.setMergeVariables(merge_variables);
+          checkEditable.setUseType(use_type);
```

Comment:
@BennyKitchell is it `setUseType`?

Reply:
@rogue0137  Yup, I would have it be `checkEditable.setUseType("operational");`